_drafts/2023-08-15-debugging-android-devices.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Debugging Android Devices
+description:
+  What should I include here... <CHRIS COME BACK AND FIX THIS!!>
```

Comment:
@bahildebrand @tyhoff - thoughts on how it should be described for SEO purposes?

Reply:
Usually a description of what you're writing keeping some keyword like "Embedded Android", "Debugging" in mind is good. Tyler usually writes great ones. His latest for an example: https://github.com/memfault/interrupt/blob/master/_posts/2023-07-26-monitoring-battery-life.md?plain=1#L3